src/app/restaurant/restaurant.component.html

```diff
@@ -1,18 +1,5 @@
-<div class="restaurant">
-  <p>Name: {{ restaurant.name }}</p>
-
-  <p>Description: {{ restaurant.description }}</p>
-
-  <p>Date of Creation: {{ restaurant.dateOfCreation }}</p>
-
-  <p>Average rating: {{ restaurant.averageRating || 0 }}</p>
-
-  <ng-container *ngFor="let star of stars">
-    <app-star-icon
-      [ngClass]="{ active: star <= restaurant.averageRating! }"
-    ></app-star-icon>
-  </ng-container>
-
+<app-restaurantcard [restaurant]="restaurant"></app-restaurantcard>
+<div class="container">
   <form [formGroup]="reviewForm" (ngSubmit)="onSubmit()">
```

Comment:
Is review form going to be a separate component as well?

Reply:
It should be done in this branch?

Reply:
No another one is ok, just checking